Unit test failure when building from release tarball

I am trying to figure out why this build is failing. 

Downloaded 0.30 for scala 2.12 from release link. I changed the hadoop version via 

```
mvn install -Dhadoop.version=3.1.0
```

```
TransformFragmentsSuite:
- load queryname sorted sam and save as parquet
- cannot sort if not saving as sam
2020-01-03 23:57:40 WARN  TransformFragments:190 - If loading and saving as alignments, consider using TransformAlignments instead
- load reads as sam and save them sorted
- bin quality scores on reads
TransformGenotypesSuite:
- save a file sorted by contig index
- save a lexicographically sorted file
- transform VCF without nested annotations
- transform VCF with nested annotations
AboutSuite:
- template variables have been replaced *** FAILED ***
  "${git.commit.id}" equaled "${git.commit.id}" (AboutSuite.scala:28)
  org.scalatest.exceptions.TestFailedException:
  at org.scalatest.Assertions.newAssertionFailedException(Assertions.scala:530)
  at org.scalatest.Assertions.newAssertionFailedException$(Assertions.scala:529)
  at org.scalatest.FunSuite.newAssertionFailedException(FunSuite.scala:1560)
  at org.scalatest.Assertions$AssertionsHelper.macroAssert(Assertions.scala:503)
  at org.bdgenomics.adam.cli.AboutSuite.$anonfun$new$1(AboutSuite.scala:28)
  at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
  at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
  at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
  at org.scalatest.Transformer.apply(Transformer.scala:22)
  at org.scalatest.Transformer.apply(Transformer.scala:20)
  at org.scalatest.FunSuiteLike$$anon$1.apply(FunSuiteLike.scala:186)
  at org.scalatest.TestSuite.withFixture(TestSuite.scala:196)
  at org.scalatest.TestSuite.withFixture$(TestSuite.scala:195)
  at org.scalatest.FunSuite.withFixture(FunSuite.scala:1560)
  at org.scalatest.FunSuiteLike.invokeWithFixture$1(FunSuiteLike.scala:184)
  at org.scalatest.FunSuiteLike.$anonfun$runTest$1(FunSuiteLike.scala:196)
  at org.scalatest.SuperEngine.runTestImpl(Engine.scala:286)
  at org.scalatest.FunSuiteLike.runTest(FunSuiteLike.scala:196)
  at org.scalatest.FunSuiteLike.runTest$(FunSuiteLike.scala:178)
  at org.scalatest.FunSuite.runTest(FunSuite.scala:1560)
  at org.scalatest.FunSuiteLike.$anonfun$runTests$1(FunSuiteLike.scala:229)
  at org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:393)
  at scala.collection.immutable.List.foreach(List.scala:392)
  at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:381)
  at org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:376)
  at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:458)
  at org.scalatest.FunSuiteLike.runTests(FunSuiteLike.scala:229)
  at org.scalatest.FunSuiteLike.runTests$(FunSuiteLike.scala:228)
  at org.scalatest.FunSuite.runTests(FunSuite.scala:1560)
  at org.scalatest.Suite.run(Suite.scala:1124)
  at org.scalatest.Suite.run$(Suite.scala:1106)
  at org.scalatest.FunSuite.org$scalatest$FunSuiteLike$$super$run(FunSuite.scala:1560)
  at org.scalatest.FunSuiteLike.$anonfun$run$1(FunSuiteLike.scala:233)
  at org.scalatest.SuperEngine.runImpl(Engine.scala:518)
  at org.scalatest.FunSuiteLike.run(FunSuiteLike.scala:233)
  at org.scalatest.FunSuiteLike.run$(FunSuiteLike.scala:232)
  at org.scalatest.FunSuite.run(FunSuite.scala:1560)
  at org.scalatest.Suite.callExecuteOnSuite$1(Suite.scala:1187)
  at org.scalatest.Suite.$anonfun$runNestedSuites$1(Suite.scala:1234)
  at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:36)
  at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:33)
  at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:198)
  at org.scalatest.Suite.runNestedSuites(Suite.scala:1232)
  at org.scalatest.Suite.runNestedSuites$(Suite.scala:1166)
  at org.scalatest.tools.DiscoverySuite.runNestedSuites(DiscoverySuite.scala:30)
  at org.scalatest.Suite.run(Suite.scala:1121)
  at org.scalatest.Suite.run$(Suite.scala:1106)
  at org.scalatest.tools.DiscoverySuite.run(DiscoverySuite.scala:30)
  at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:45)
  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13(Runner.scala:1349)
  at org.scalatest.tools.Runner$.$anonfun$doRunRunRunDaDoRunRun$13$adapted(Runner.scala:1343)
  at scala.collection.immutable.List.foreach(List.scala:392)
  at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1343)
  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24(Runner.scala:1033)
  at org.scalatest.tools.Runner$.$anonfun$runOptionallyWithPassFailReporter$24$adapted(Runner.scala:1011)
  at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1509)
  at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:1011)
  at org.scalatest.tools.Runner$.main(Runner.scala:827)
  at org.scalatest.tools.Runner.main(Runner.scala)
- templated values are not empty
CoverageSuite:
- correctly calculates coverage from small sam file
MergeShardsSuite:
- merge shards from unordered sam to unordered sam
- unordered sam to ordered sam
- merge sharded bam
- merge sharded cram
TransformAlignmentsSuite:
- unordered sam to unordered sam
- unordered sam to ordered sam
- unordered sam, to adam, to sam
- unordered sam, to adam, to ordered sam
- put quality scores into bins
- run region predicate
ADAMMainSuite:
- default command groups is non empty
- module provides default command groups
- inject default command groups when called via main
- command groups is empty when called via apply
- single command group
- add new command group to default command groups
- module restores default command groups when called via apply
- custom module with single command group
- custom module with new command group added to default command groups
Adam2FastqSuite:
2020-01-03 23:57:49 WARN  RDDBoundAlignmentDataset:190 - Found 16 read names that don't occur exactly twice:
	1x:	16

Samples:
	SRR062634.16445865
	SRR062634.9119161
	SRR062634.17190076
	SRR062634.17969132
	SRR062634.7301099
	SRR062634.2087100
	SRR062634.20911784
	SRR062634.16769670
	SRR062634.18958430
	SRR062634.12099057
	SRR062634.12606172
	SRR062634.14985224
	SRR062634.10448889
	SRR062634.4789722
	SRR062634.3203184
	SRR062634.17698657
- convert SAM to paired FASTQ
Run completed in 21 seconds, 161 milliseconds.
Total number of tests run: 44
Suites: completed 12, aborted 0
Tests: succeeded 43, failed 1, canceled 0, ignored 0, pending 0
*** 1 TEST FAILED ***
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for ADAM_2.12 0.30.0:
[INFO] 
[INFO] ADAM_2.12 .......................................... SUCCESS [  4.980 s]
[INFO] ADAM_2.12: Shader workaround ....................... SUCCESS [  3.270 s]
[INFO] ADAM_2.12: Avro-to-Dataset codegen utils ........... SUCCESS [  4.190 s]
[INFO] ADAM_2.12: Core .................................... SUCCESS [06:19 min]
[INFO] ADAM_2.12: APIs for Java, Python ................... SUCCESS [ 18.977 s]
[INFO] ADAM_2.12: CLI ..................................... FAILURE [ 22.935 s]
[INFO] ADAM_2.12: Assembly ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  07:13 min
[INFO] Finished at: 2020-01-03T23:57:51-08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.scalatest:scalatest-maven-plugin:2.0.0:test (test) on project adam-cli-spark2_2.12: There are test failures -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.scalatest:scalatest-maven-plugin:2.0.0:test (test) on project adam-cli-spark2_2.12: There are test failures
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: There are test failures
    at org.scalatest.tools.maven.TestMojo.execute (TestMojo.java:107)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :adam-cli-spark2_2.12

```

--- a/adam-cli/src/test/scala/org/bdgenomics/adam/cli/AboutSuite.scala
+++ b/adam-cli/src/test/scala/org/bdgenomics/adam/cli/AboutSuite.scala
@@ -25,7 +25,6 @@ class AboutSuite extends FunSuite {
   test("template variables have been replaced") {
     assert(about.artifactId !== "${project.artifactId}")
     assert(about.buildTimestamp !== "${maven.build.timestamp}")
-    assert(about.commit !== "${git.commit.id}")
     assert(about.hadoopVersion !== "${hadoop.version}")
     assert(about.scalaVersion !== "${scala.version}")
     assert(about.sparkVersion !== "${spark.version}")
@@ -35,7 +34,6 @@ class AboutSuite extends FunSuite {
   test("templated values are not empty") {
     assert(about.artifactId.nonEmpty)
     assert(about.buildTimestamp.nonEmpty)
-    assert(about.commit.nonEmpty)
     assert(about.hadoopVersion.nonEmpty)
     assert(about.scalaVersion.nonEmpty)
     assert(about.sparkVersion.nonEmpty)

